ext-src/worker.ts

```diff
@@ -158,35 +162,48 @@ process.on("message", async m => {
   var meta = new grpc.Metadata();
   meta.add('authorization', m.jwtToken);
   if (m.command === "compile") {
```

Comment:
in `process.on("message", async m => {` all the if checks are on `m.command`. use `switch-case` instead.

Reply:
```js
process.on("message", async m => {
  var meta = new grpc.Metadata();
  meta.add('authorization', m.jwtToken);
  switch (m.command) {
    case "compile":
      compileFn();
  }
```

This can be alternate `switch-case` implementation. But there are multiple functions and conditions. Need to evaluate what improvement that brings us. But all the functions in the conditions can be moved to individual functions.
@diptajitbasak create an issue for this